src/address/index.ts

```diff
@@ -315,6 +347,13 @@ export class AddressDelegated extends Address {
 
     return new AddressDelegated(network, namespace, subAddress)
   }
+
+  toEthAddressHex = (): String => {
```

Comment:
don't you think this one should not be in the AddressDelegated class. It is a generic class for delegated address. We could move it to FilEthAddress class instead.

Reply:
I don't know what to do with the `FilEthAddress` because it is an extend to AddressDelegated but this address can also be an AddressID.